QuickDeployment/QuickDeploymentEnvironment.md

```diff
@@ -8,63 +8,62 @@ This is an overview on the Chocolatey Quick Deployment Environment (QDE). It pro
+- **Jenkins** (pre-configured with internalizer jobs) - This is your automation engine, that lets you run tasks on-demand and on a schedule. It will host the PowerShell scripts to help you auto-internalize packages from the Chocolatey Community Repository
+- **Chocolatey Central Management** - This is your web dashboard for Chocolatey, that will allow you to track and monitor Chocolatey packages on your endpoint clients. You can see what packages are installed where, and whether or not they are out-of-date.
+- **Scripts for Internal Deployment** - Various scripts to help you configure this solution are included, for your convenience.
 
 > :warning: **WARNING**: This solution targets environments up to about **1000 nodes**.
-If you need a solution for a larger environment, QDE would still be suitable as a proof-of-concept, however, best practices would recommmend a distributed infrastructure, separating each component into its own discrete node.
-If you find yourself in need of a more scalable solution, please contact Support and we'll be more than happy to provide guidance for larger solutions.
+> If you need a solution for a larger environment, QDE would still be suitable as a proof-of-concept, however, best practices would recommend a distributed infrastructure, separating each component into its own discrete node.
+> If you find yourself in need of a more scalable solution, please contact Support and we'll be more than happy to provide guidance for larger solutions.
 
 ## Links
 
-* [[Quick Deployment Environment Setup|QuickDeploymentSetup]]
-* Setting up proper DNS on QDE (DHCP by default; you'll want to switch to static)
-* [[QDE Desktop ReadMe File|QuickDeploymentDesktopReadme]] (included here for convenience)
-* [[QDE SSL/TLS Setup|QuickDeploymentSslSetup]]
-* [[QDE Firewall Changes|QuickDeploymentFirewallChanges]]
-* [[QDE Client Setup|QuickDeploymentClientSetup]] (setting up your client machines)
-* Opening up for remote access (for QDE)
+- [[Quick Deployment Environment Setup|QuickDeploymentSetup]]
+- [[QDE Desktop ReadMe File|QuickDeploymentDesktopReadme]] (included here for convenience)
+- [[QDE SSL/TLS Setup|QuickDeploymentSslSetup]]
+- [[QDE Firewall Changes|QuickDeploymentFirewallChanges]]
+- [[QDE Client Setup|QuickDeploymentClientSetup]] (setting up your client machines)
 
 ## Related Articles
 
 A lot of what is done in QDE compresses the work or completely removes the work found in these related articles.
 
-* [[Organizational Deployment Guide|How-To-Setup-Offline-Installation]]
-* [Client Install](https://chocolatey.org/install#organization)
-* [[Chocolatey Commercial Installation|Installation-Licensed]]
-* [[Automate Package Internalization|How-To-Setup-Internal-Package-Repository]]
+- [[Organizational Deployment Guide|How-To-Setup-Offline-Installation]]
+- [[Client Install|QuickDeploymentClientSetup]]
```

Comment:
Incorrect. Put it back the way it was.

Reply:
It's a reference to Chocolatey Installation for organizations and should NOT point to anything QDE. Update the wording if you must to "Chocolatey Installation"

Reply:
That'll be a bit clearer for folks on first glance, I agree. Thanks!